Makefile

```diff
@@ -819,6 +819,7 @@ PREVIOUS_TAG ?= $(shell git tag -l | grep -E "^v[0-9]+\.[0-9]+\.[0-9]+$$" | sort
 RELEASE_ALIAS_TAG := $(PULL_BASE_REF)
 RELEASE_DIR := out
 RELEASE_NOTES_DIR := _releasenotes
+# can be set directly if needed, e.g., export USER_FORK=<personal GitHub handle>
```

Comment:
```suggestion
```

I think this is redundant - as it's true for environmental variables in general.

Reply:
Moved to the release-tasks.md as suggested, thanks!